format/src/main/thrift/carbondata.thrift

```diff
@@ -1,223 +1,226 @@
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ *    http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
  */
 
 /**
- * File format description for the carbon file format
+ * File format description for the CarbonData file format
  */
 namespace java org.apache.carbondata.format
 
 include "schema.thrift"
 include "dictionary.thrift"
 
 /**
-* Information about a segment, that represents one data load
-*/
+ * Information about a segment, that represents one data load
+ */
 struct SegmentInfo{
-    1: required i32 num_cols; // Number of columns in this load, because schema can evolve . TODO: Check whether this is really required
+    1: required i32 num_cols; // Number of columns in this load, because schema can evolve, different segments may have different columns
     2: required list<i32> column_cardinalities; // Cardinality of columns
 }
 
 /**
-*	Btree index of one blocklet
-*/
+ * Btree index of one blocklet
+ */
 struct BlockletBTreeIndex{
     1: required binary start_key; // Bit-packed start key of one blocklet
-    2: required binary end_key;	// Bit-packed start key of one blocklet
+    2: required binary end_key;	// Bit-packed end key of one blocklet
 }
 
 /**
-*	Min-max index of one blocklet
-*/
+ * Min-max index of one blocklet
+ */
 struct BlockletMinMaxIndex{
     1: required list<binary> min_values; //Min value of all columns of one blocklet Bit-Packed
     2: required list<binary> max_values; //Max value of all columns of one blocklet Bit-Packed
 }
 
 /**
-* Index of one blocklet
-**/
+ * Index of one blocklet
+ */
 struct BlockletIndex{
     1: optional BlockletMinMaxIndex min_max_index;
     2: optional BlockletBTreeIndex b_tree_index;
 }
 
 /**
-* Sort state of one column
-*/
+ * Sort state of one column
+ */
 enum SortState{
     SORT_NONE = 0; // Data is not sorted
     SORT_NATIVE = 1; //Source data was sorted
     SORT_EXPLICIT = 2;	// Sorted (ascending) when loading
 }
 
 /**
-*	Compressions supported by Carbon Data.
-*/
+ * Compressions supported by CarbonData.
+ */
 enum CompressionCodec{
     SNAPPY = 0;
 }
 
 /**
-* Represents the data of one dimension one dimension group in one blocklet
-*/
-// add a innger level placeholder for further I/O granulatity
+ * Represents the data of one column page or one column page group in one blocklet.
```

Comment:
Better phrase as follow.
`Represents the data of one column page or one column page group in side blocklet.`